Resources/Prototypes/DeltaV/Roles/Jobs/Command/admin_assistant.yml

```diff
@@ -0,0 +1,75 @@
+- type: job
+  id: AdministrativeAssistant
+  name: job-name-admin-assistant
+  description: job-description-admin-assistant
+  playTimeTracker: JobAdminAssistant
+  requirements:
+    - !type:DepartmentTimeRequirement
+      department: Logistics
+      time: 18000 # 5 hours
+    - !type:DepartmentTimeRequirement
+      department: Engineering
+      time: 18000 # 5 hours
+    - !type:DepartmentTimeRequirement
+      department: Medical
+      time: 18000 # 5 hours
+    - !type:DepartmentTimeRequirement
+      department: Security
+      time: 18000 # 5 hours
+    - !type:DepartmentTimeRequirement
+      department: Epistemics 
+      time: 18000 # 5 hours
+    - !type:DepartmentTimeRequirement
```

Comment:
Common consensus seems to suggest that people think more than 10ish hours for requirement is absurd, might be worth reconsidering this.